src/clip.sol

```diff
@@ -210,11 +210,11 @@ contract Clipper {
     // Could get this from rmul(Vat.ilks(ilk).spot, Spotter.mat()) instead, but
     // if mat has changed since the last poke, the resulting value will be
     // incorrect.
-    function getPrice() internal returns (uint256 price) {
+    function feed() internal returns (uint256 feed) {
```

Comment:
There is a shadowed variable warning here. I'd go with the other PR which is much clearer what is the value.

Reply:
I mean, the shadowing is easy to resolve...just don't use a declared return var with the same name as the function.

Reply:
Yeah, but also not sure if you can do `uint256 feed = feed();`.

Reply:
If it's the case you could solve it with `uint256 feed_ = feed();` though

Reply:
Whatever you prefer, if we find good daiwanese names for distinguishing both type of prices I'm ok. Just don't want to spend much time on it. I'm not sure if `mark` says much to me. IMO in this type of cases where we have two completely different values but with a very similar meaning is difficult to use daiwanese.